src/fobnail-attester.c

```diff
@@ -10,13 +10,18 @@
 #include <signal.h>
 #include <stdbool.h>
 
+#include <meta.h>
+
 static bool quit = false;
 static const char LISTEN_ADDRESS[] = "0.0.0.0";
 static unsigned int port = COAP_DEFAULT_PORT; /* default port 5683 */
 
 static void handle_sigint(int signum)
 {
-    quit = true;
+    switch(signum)
+    case SIGTERM:
+    default:
+        quit = true;
```

Comment:
@0xDen
- If you want to handle multiple signals please change function name to something more general like `handle_signal`.
- You forgot to attach handler to SIGTERM.
- IMO this `switch` is redundant as regardless of signal type the same thing will happen.

Reply:
Fixed